### DISCUSSION TITLE
SQLAlchemy defaults are not present in Pydantic model

### DISCUSSION THREAD

**Discussion by martinhanzik** [tiangolo/pydantic-sqlalchemy#86] [Questions] [answered]
```
class TestSchema(Base):
    __tablename__ = 'test'

    id = Column(INTEGER(10, unsigned=True), primary_key=True)
    active = Column(Boolean, default=True, nullable=False)

Test = sqlalchemy_to_pydantic(TestSchema)

Test(id=1)
# Test(id=1)
Test(id=1).dict()
# TestSchema(id=1, active=None)
```

`active` should be True

**Comment by M-Qi** [comment]
是的如果需要这个默认值支持，就需要该源码

**Comment by tiangolo** [comment, marked as answer]
Thanks for the patience with my reply! :sweat_smile:

Yep, there's no support for many things in both SQLAlchemy and Pydantic here. :melting_face: 

I deprecated this library a while ago in favor of SQLModel which would solve all the same use cases in a much better way, and I'm directing all the effort in that direction.

SQLModel uses Pydantic models to define both data and table (SQL) models, so it's easier to do all these things. :rocket:
